### PR TITLE
add mitoz package for IB134L

### PR DESCRIPTION
**Pull request by petersudmant**
update biology hub for IB134L to support the mitoz package for next week's tutorial.

### CODE REVIEW

**Comment by petersudmant** [comment]
@yuvipanda is it possible for me to have permission to merge changes? It would allow us to update material for students rapidly which would be very helpful! Thank you!

**Comment by yuvipanda** [comment]
@petersudmant I've invited you to the org, which should give you merge rights. Can you join the #ucb-datahubs channel on uctech.slack.com to co-ordinate with the rest of us?

This also gives you access to grafana.datahub.berkeley.edu :)

**Comment by yuvipanda** [comment]
@petersudmant I'd also suggest using http://nbgitpuller.link for content, using this only for environment changes. Looks like that's what you're doing anyway, but wanted to explicitly say :)

File: deployments/biology/image/postBuild [hunk 1]
```diff
@@ -10,4 +10,30 @@ cp ipython_config.py ${CONDA_DIR}/etc/ipython/ipython_config.py
 
 wget http://phylosolutions.com/paup-test/paup4a168_ubuntu64.gz -O ${CONDA_DIR}/bin/paup.gz
 gunzip ${CONDA_DIR}/bin/paup.gz
-chmod +x ${CONDA_DIR}/bin/paup
\ No newline at end of file
+chmod +x ${CONDA_DIR}/bin/paup
+
+# mitoZ installation
```
Comment (yuvipanda) [inline deployments/biology/image/postBuild:15]: Can you make a note here about what class this is for?

**Pull request merged**